projects/observability/src/shared/components/cartesian/chart.ts

```diff
@@ -54,6 +54,9 @@ export interface Series<TInterval> {
   getTooltipTitle?(datum: TInterval): string;
 }
 
+interface SeriesGroupBy extends FilterAttributeExpression {
+  metadata: FilterAttribute[];
```

Comment:
this isn't metadata specified to the series right? can you name it allMetadata or something?

Reply:
renamed